Modules/comm.lua

```diff
@@ -1096,3 +400,1125 @@ function CommDKP.Sync:DeserializeStringToTable(_string)
   end
 
 end
+
+
+----------
+-- FULL BROADCAST HANDLERS
+----------
+
+function FullBroadcastSend(prefix, commObject, channel)

```

Comment:
This name is a little too general for a global function. Should be given a CommDKP prefix, stored in a global table, or localized.
Same with FullBroadcastReceived (423), TableMergeSend (539), and TableMergeReceived (553)

Reply:
Cleaned up the function names (all of em) by adding CommDKP: prefix to them